### PR TITLE
Succumb Fix

### PR DESCRIPTION
**Pull request by Geevies**
* Succumbing no longer needs massive brain damage to work, it is much easier to kill yourself.

### CODE REVIEW

**Comment by Geevies** [comment]
Brainmed made it need massive brain damage. This made it need somewhere in the range of medium crit.

**Comment by Geevies** [comment]
!bugfix

**Comment by Geevies** [comment]
!review

File: code/modules/mob/living/carbon/human/human.dm [hunk 1]
```diff
@@ -1218,6 +1218,15 @@
 		remoteview_target = null
 		reset_view(0)
 
+/mob/living/carbon/human/succumb()
+	set hidden = TRUE
+
+	if(shock_stage > 50)
```
Comment (NonQueueingMatt) [inline code/modules/mob/living/carbon/human/human.dm]: This is way too light of a condition.

File: code/modules/mob/living/carbon/human/human.dm [hunk 2]
```diff
@@ -1218,6 +1218,15 @@
 		remoteview_target = null
 		reset_view(0)
 
+/mob/living/carbon/human/succumb()
+	set hidden = TRUE
+
+	if(shock_stage > 50 && (getBruteLoss() > 40 || getFireLoss() > 40 || getOxyLoss() > 40 || getToxLoss() > 40))
```
Comment (NonQueueingMatt) [inline code/modules/mob/living/carbon/human/human.dm:1224]: Use get_total_health() at this point.

Comment (Geevies) [reply code/modules/mob/living/carbon/human/human.dm:1224]: kk

File: code/modules/mob/living/carbon/human/human.dm [hunk 3]
```diff
@@ -1218,6 +1218,15 @@
 		remoteview_target = null
 		reset_view(0)
 
+/mob/living/carbon/human/succumb()
+	set hidden = TRUE
+
+	if(shock_stage > 50 && (maxhealth * 0.6) > get_total_health())
```
Comment (Geevies) [inline code/modules/mob/living/carbon/human/human.dm:1224]:
```suggestion
	if(shock_stage > 50 && (maxHealth * 0.6) > get_total_health())
```

**Pull request merged**